google-cloud-datastore/pom.xml

```diff
@@ -16,6 +16,7 @@
   </parent>
   <properties>
     <site.installationModule>google-cloud-datastore</site.installationModule>
+    <error-prone.version>2.10.0</error-prone.version>
```

Comment:
is there any particular reason why this is declared in these two modules? It would be cleaner to declare this in the parent and have each module inherit and run the config: https://github.com/googleapis/java-datastore/blob/main/pom.xml

Reply:
to be clear, I am referring to both the property definition as well as plugin definition.

Reply:
Honestly, I was afraid it would affect the ability to run `mvn` in the child directory and I didn't know whether you wanted to preserve that behavior. I can centralize these settings.

Reply:
Typically all CI is run from the parent, so let's try centralizing it for consistency and see if we run into any issues.

Reply:
Done.